12-stretch-slim/Dockerfile

```diff
@@ -0,0 +1,62 @@
+# Debian-based image maintained by Node, see https://hub.docker.com/_/node/ for a full list of available images.
+# Source for this base image can be found at https://github.com/nodejs/docker-node/tree/master/12/stretch-slim
+FROM node:12-stretch-slim
+
+# Let's update and install the things.
+# Run apt-get quietly (-qq) and say yes to prompts (-y)
+# See best practices for apt-get in Docker at https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run
+# Note: we install Imagemagick in our base image so we can keep it updated in one place, and so we can add a strict security policy by default (see further down)
+# Note: we install python 2 because node-gyp v5 does not support python 3, & we want child images to continue to be able to build native addons
+RUN apt-get update -qq \
+    && apt-get upgrade -y \
+    && apt-get install -y build-essential imagemagick python2.7 \
```

Comment:
We don't want python 2 anymore. See #57

Reply:
shall we merge that and I'll rebase and incorporate it into the the 12 image?

Reply:
🤷‍♂

Reply:
I went ahead and removed it.